.buildkite/build-static-release.sh

```diff
@@ -46,29 +46,51 @@ popd
 pushd gems/sorbet
 sed -i.bak "s/0\\.0\\.0/${release_version}/" sorbet.gemspec
 gem build sorbet.gemspec
+
+# This part tests the actual compiled gem (outside of snapshot harness).
+# We can't test this inside of the test harness, because we'd want to run it
+# via `bundle exec srb init`, which would require that each Gemfile in our test
+# suite list `sorbet` as a dependency. This interacts poorly with local
+# development, because then to even run the tests locally you'd have to build
+# the gem (which involves building sorbet-static and sorbet).
+#
+# Thus, we only have these two smoke tests of `srb init` via the whole gem.
+# This is to catch environment-specific things, like when making changes to the
+# way that the `sorbet-static` libexec/ folder is discovered. Most tests should
+# be possible to be written as normal snapshot tests.
+
+# TODO(jez) It's not clear to me why we need these here for the tests to run.
+# We should investigate how to not need these dependencies, because `srb init`
+# doesn't depend on them except in development.
+rbenv exec gem install minitest mocha
```

Comment:
I think this is related to the Gemfile in the parent directory referring to the gemspec. Since we use `Bundler.load` unguarded in `fetch-rbis.rb`, it will find this Gemfile, and require that those libraries be present.